sklearn/naive_bayes.py

```diff
@@ -354,11 +363,13 @@ def _partial_fit(self, X, y, classes=None, _refit=False,
             n_classes = len(self.classes_)
             self.theta_ = np.zeros((n_classes, n_features))
             self.sigma_ = np.zeros((n_classes, n_features))
+            # create a 2d-array of uncorrected variances for further use
+            # in _update_mean_variance()
+            self.sigma_uncorrected_ = np.zeros((n_classes, n_features))
```

Comment:
why not using the same trick as before (cf below) 

# put epsilon back each time?

Reply:
Because we don't use a constant epsilon anymore.

Reply:
And we can't subtract a min_variance from self.sigma_ (that way we'd always get zero calculating previous sigma).